src/codeu/chat/server/Server.java

```diff
@@ -247,47 +251,45 @@ private void onBundle(Relay.Bundle bundle) {
     User user = model.userById().first(relayUser.id());
 
     if (user == null) {
-      user = controller.newUser(relayUser.id(), relayUser.text(), relayUser.time());
+      user = controller.newUser(relayUser.id(), relayUser.text(),
+          relayUser.time());
     }
 
-    ConversationHeader conversation = model.conversationById().first(relayConversation.id());
+    ConversationHeader conversation = model.conversationById()
+        .first(relayConversation.id());
 
     if (conversation == null) {
 
-      // As the relay does not tell us who made the conversation - the first person who
-      // has a message in the conversation will get ownership over this server's copy
+      // As the relay does not tell us who made the conversation - the first
+      // person who
```

Comment:
Even after changing the default line wrap to 100, the formatter keeps trying to split it into three different lines...